src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -348,18 +343,24 @@ exports.createFiberFromText = function(
 };
 
 function createFiberFromElementType(
-  type: mixed,
-  key: null | string,
+  element: ReactElement,
```

Comment:
Seems like there's no point in this being a separate function anymore. Let's inline it into `createFiberFromElement`.